components/ai_chat/resources/page/components/code_block/index.tsx

```diff
@@ -4,17 +4,16 @@
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
-
-import styles from './style.module.scss'
-import { getLocale } from '@brave/brave-ui'
-import Button from '@brave/leo/react/button'
-import Icon from '@brave/leo/react/icon'
 import { Light as SyntaxHighlighter } from 'react-syntax-highlighter'
 import hljsStyle from 'react-syntax-highlighter/dist/esm/styles/hljs/ir-black'
 import cpp from 'react-syntax-highlighter/dist/esm/languages/hljs/cpp'
 import javascript from 'react-syntax-highlighter/dist/esm/languages/hljs/javascript'
 import python from 'react-syntax-highlighter/dist/esm/languages/hljs/python'
 import json from 'react-syntax-highlighter/dist/esm/languages/hljs/json'
+import Button from '@brave/leo/react/button'
```

Comment:
ah, I changed one of the imports because there's no reason to be importing from `brave-ui` so I cleaned up the very-wrong order. But I moved them a little too far. Fixing